src/Symfony/Component/Notifier/Bridge/Brevo/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+CHANGELOG
+=========
+
+6.3
+---
+
+* Added the bridge
```

Comment:
should you mentioned that this is the replacement of https://packagist.org/packages/symfony/sendinblue-notifier?
and also in the other composer, mark it as deprecated?